ui/components/remove.js

```diff
@@ -0,0 +1,32 @@
+import DeleteModal from './delete-modal'
+
+export default function Remove({
```

Comment:
Not sure we want to couple the button + modal here since they are quite different – would it be better to leave them separate? – or better have the sidebar layout include these?

Reply:
this is the remove functionality on the sidebar. In my opinion i think it is ok because the modal only appear when the button clicks